src/expressions/mvOverlapExpression.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2016-2020 Imply Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { PlywoodValue } from '../datatypes/index';
+import { SQLDialect } from '../dialect/baseDialect';
+
+import {
+  ChainableUnaryExpression,
+  Expression,
+  ExpressionJS,
+  ExpressionValue,
+} from './baseExpression';
+
+export class MvOverlapExpression extends ChainableUnaryExpression {
+  static op = 'MvOverlap';
+  static fromJS(parameters: ExpressionJS): MvOverlapExpression {
+    const value = ChainableUnaryExpression.jsToValue(parameters);
+    return new MvOverlapExpression(value);
+  }
+
+  constructor(parameters: ExpressionValue) {
+    super(parameters, dummyObject);
+    this._checkOperandTypes('STRING');
+    this._checkExpressionTypes('STRING');
```

Comment:
```suggestion
    this._checkOperandTypes('SET/STRING');
    this._checkExpressionTypes('SET/STRING');
```

Per the docs, I think both of these are supposed to be arrays?

> Returns 1 if arr1 and arr2 have any elements in common, else 0.